recipes/flit/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+# Build might be run as root, but it doesn't matter
+set -e
+
+export FLIT_ROOT_INSTALL=1
+python -m flit install --env --deps none
```

Comment:
In theory `python` should do the right thing. using the env var is just to be on the safe side. (I am updating the example recipe.)